openshift-templates/hawkular-apm-server.yml

```diff
@@ -8,8 +8,8 @@ metadata:
     description: Hawkular APM Server service.
     iconClass: icon-wildfly
     openshift.io/display-name: Hawkular APM Server
-    tags: instant-app,hawkular,apm,opentracing
-  name: hawkular-apm-server
+    tags: instant-app,hawkular,apm,opentracing,quickstart
```

Comment:
Not sure having 'quickstart' as a tag is a good idea, as it could imply and example rather than an actual service?